packages/lib/src/listing-io.ts

```diff
@@ -0,0 +1,78 @@
+import { isIndexArgument } from './api-command'
+import { formatAndWriteItem, ListDataFunction, LookupDataFunction, CommonOutputProducer } from './basic-io'
+import { listTableFormatter, sort, writeOutput } from './output'
+import { buildOutputFormatter } from './output-builder'
+import { SmartThingsCommand } from './smartthings-command'
+import { TableFieldDefinition } from './table-generator'
+
+
+// TODO: rename and maybe export
```

Comment:
`translateStringToId`?

We can always export if/when we need it, so I wouldn't now if it isn't needed.

Reply:
I had left this in because I figured I would deal with it when we are at the end and the one with the conflicting name from `io-command.ts` is gone and we can use `stringTranslateToId`.

`translateStringToId` does read nicer but `stringTranslateToId` feels more precise to me because it is the `string`-specific version of `translateToId`. If you have strong feelings though, I either name works for me.